eva-peng/week7/tues/api-axios-homework/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AJAX Intro</title>
+    <script src="js/jquery.js"></script>
+    <script src="js/axios.js"></script>
+    <script src="js/main.js"></script>
+    <style>
+        body {
+          background-image: url('https://www.citybikeadventures.com/front/images/berlin/berlin1.png');
+          background-repeat: no-repeat;
+        }
+
+        </style> 
+</head>
+<body>
+    <h1>Search A Bike</h1>
+
+    <input type="text" id="searchQuery" placeholder="enter a name">
+    <button id="submitButton">Lookup a CityBike Name</button>
+    <div id="results"></div>
```

Comment:
it might be worth explaining how to use this API. I have never heard of it, and definitely don't know what it does. 
I had to look at the json data to determine what it was. 

Maybe change it around and see what each city provides